packages/core/src/organization-contact/commands/handlers/organization-contact-update.handler.ts

```
@@ -34,7 +34,16 @@ export class OrganizationContactUpdateHandler implements ICommandHandler<Organiz
 				});
 			} catch (error) {
-				console.log('Error occurred during creation of contact details or creating the organization contact:', error);
+				console.log(
+					'Error occurred during creation of contact details or creating the organization contact:',
+					error
+				);
+			}
+
+			if (isNotEmpty(input.projects)) {
+				input.projects.forEach((project) => {
+					delete project.members;
```

Biome

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)